pnet_packet/Cargo.toml

```diff
@@ -16,5 +16,15 @@ pnet_base = { path = "../pnet_base", version = "0.29.0" }
 pnet_macros_support = { path = "../pnet_macros_support", version = "0.29.0" }
 pnet_macros = { path = "../pnet_macros", version = "0.29.0" }
 
+[dev-dependencies]
+hex = "0.4.3"
+criterion = {version = "0.3.5", features = ["html_reports"]} #added HTML feature becuase of the annoying warnings when running the tests
+pprof = { version = "0.7", features = ["flamegraph"] }
+
 [build-dependencies]
 glob = "0.3.0"
+
+[[bench]]
```

Comment:
I think these changes were in the other PR? Might need a rebase for github to pick it up?

Similarly for the other benchmarking stuff here.

Reply:
done!